packages/service/src/shims/workspace.ts

```diff
@@ -83,7 +83,7 @@ export class WorkspaceShimService extends Disposable {
 
   $getDocumentByLspUri(uri: lsp.URI): TextDocument | undefined {
```

Comment:
```suggestion
$getDocumentByLspUri(uri: lsp.URI) {
    return this._documents.get(URI.parse(uri));
}
```

Reply:
Thanks for catching this! Could you try if this fix works for you? Replacing the whole method should be better for this case.

Reply:
@yioneko 
it works well.